components/browser/errorpages/src/main/assets/parseQueryAndInject.js

```diff
@@ -0,0 +1,37 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+/**
+ * Handles the parsing of the ErrorPages URI and then passes them to injectValues
+ */
+window.parseQuery = function(queryString) {
+   var query = {};
+   var pairs = (queryString[0] === '?' ? queryString.substr(1) : queryString).split('&');
+   for (var i = 0; i < pairs.length; i++) {
+     var pair = pairs[i].split('=');
+     query[decodeURIComponent(pair[0])] = decodeURIComponent(pair[1] || '');
+   }
+   injectValues(query)
+ };
+
+/**
+ * Updates the HTML elements based on the queryMap
+ */
+function injectValues(queryMap) {
+   // Go through each element and inject the values :)
+   document.title = queryMap.title
+   document.getElementById('errorTitleText').innerHTML = queryMap.title
+   document.getElementById('errorShortDesc').innerHTML = queryMap.description
+   document.getElementById('errorTryAgain').innerHTML = queryMap.button
+
+   // If no image is passed in, remove the element so as not to live an empty iframe
+   if (queryMap.image == "") {
```

Comment:
`==` is loose equality in JS, and I think any falsey value will match here (false, null, undefined). If that's desired behaviour maybe just writing as "not" is better.
```suggestion
   if (!queryMap.image) {
```

Reply:
👍